recipes/iphop/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 
 source:
   url: https://bitbucket.org/srouxjgi/iphop/downloads/{{ name }}-{{ version }}.tar.gz
-  sha256: 0c31798e80132477ec220cd8ace552199978cf27160ca0f450462291981e3b53
+  sha256: d61ef488175f1a99784d48a9a8d98c40959a3f49d6129ad11e5ace81b08be4f5
```

Comment:
Can you create a new release instead of changing the old one? Changing the license and adding new deps looks like a justified change for a new version.

Reply:
So the version 1.1 is supposed to be this one with the new license and the bug fixes. What was pushed to BioConda as 1.1 a few days ago (by someone else, and I think by mistake), was an intermediary non-stable version that still had some issues and bugs, and should not work anymore (it points to a file that does not exist anymore). I'm not sure what the best course of action is: I understand that overriding this 1.1 is not optimal, but bumping this pull request as 1.2 would also leave in place a non-functioning version 1.1. let me know what you think. Thanks!